setup_schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org",
+    "$id": "http://yio-remote.com/yiodock.json",
+    "type": "object",
+    "title": "YIO Dock Integration Schema",
+    "description": "Required data points to set up a YIO Dock integration.",
+    "default": {},
+    "additionalProperties": true,
+    "examples": [
+        {
+            "ip": ""
```

Comment:
An example should contain a value ;-)

Reply:
Wait, there's also properties.ip with an example. I think this examples section is not required

Reply:
Right!